app/src/main/java/com/imobile3/groovypayments/ui/main/MainDashboardActivity.java

```diff
@@ -102,7 +104,17 @@ private void handleDashboardButtonClick(@NonNull MainDashboardButton button) {
                 break;
 
             case Placeholder2:
-                throw new RuntimeException("User clicked a Placeholder button");
+                //throw new RuntimeException("User clicked a Placeholder button");
```

Comment:
remove this line don't comment it out, also add a screenshot of the dialog in the description of the PR when completed